src/modules/displayDOM.js

```diff
@@ -0,0 +1,18 @@
+import getScore from './getScores.js';
+
+const displayDOM = async () => {
+  let output = '';
+  const scoreList = document.querySelector('.score-list');
+  const scores = await getScore();
+  scores.result.forEach((score, index) => {
+    output += `
+      <li class="score-line">
+      <span id="username">${scores.result[index].user}</span>  
+      <span id="userscore">${scores.result[index].score} wins </span>
```

Comment:
- [ ] **[OPTIONAL]**  You could improve this piece of code by actually using the `score` variable you defined inside the forEach loop:

```suggestion
  scores.result.forEach((score) => {
    output += `
      <li class="score-line">
      <span id="username">${score.user}</span>  
      <span id="userscore">${score.score} wins </span>
```

remember that using `forEach` will give you access to each element in the array, since those elements are objects with `user` and `score` properties, you can access directly to these properties instead of looking at them in the original array! 🤓 🔧

Reply:
Alright. I have also fixed that.